doc/source/NewSiteConfigs.rst

```diff
@@ -258,9 +258,15 @@ Remember to activate the ``lua`` module environment and have MacTeX in your sear
 
 10. Process the specs and install
 
+It is recommended to save the output of concretize in a log file and inspect that log file using the show_duplicate_packages.py utility.
+This is done to find and fix duplicate package specifications before running the install and discovering the duplicate packages hours later.
+There a few acceptable (i.e., by design) duplicate packages which are: crtm, fms, mapl and esmf.
```

Comment:
Are we still maintaining duplicates of mapl and esmf?

Reply:
Based on the unified-dev template it doesn't look like we're adding the extra esmf/mapl versions anymore. Alternatively, we could generalize this to something along the lines of "Note that in the unified environment, there may be deliberate duplicates; consult the specs in spack.yaml to determine which ones are desired."

Reply:
Great idea! I've made these changes. Thanks